kpops/cli/main.py

```diff
@@ -175,11 +175,14 @@ def log_action(action: str, pipeline_component: PipelineComponent):
 
 
 def create_pipeline_config(
-    config: Path, defaults: Path, verbose: bool
+    config: Path, defaults: Optional[Path], verbose: bool
 ) -> PipelineConfig:
     setup_logging_level(verbose)
     PipelineConfig.Config.config_path = config
-    pipeline_config = PipelineConfig(defaults_path=defaults)
+    if defaults:
+        pipeline_config = PipelineConfig(defaults_path=defaults)
+    else:
+        pipeline_config = PipelineConfig()
```

Comment:
wouldn't it be easier to do "calculate defaults path relative to config.yaml" here? saves the hassle of working with dict keys and such

Reply:
I first tried that, but for some reason I kept getting errors, I couldn't extract `defaults_path` from the `PipelineConfig` object and decided not to waste more time and to at least get a working solution first. I could try it again after I write the unit test.